okio/src/main/java/okio/RealBufferedSource.java

```diff
@@ -233,6 +233,16 @@ public RealBufferedSource(Source source) {
     return buffer.readLongLe();
   }
 
+  @Override public long readDecimalLong() throws IOException {
+    request(20);
```

Comment:
I think we need something like `RealBufferedSource.indexOfElement()`, but actually inverted. Maybe a private method, since otherwise the name will be really awkward?